app/(gcforms)/[locale]/(app administration)/admin/(with nav)/flags/page.tsx

```diff
@@ -23,6 +25,10 @@ export async function generateMetadata(props: {
 export default AuthenticatedPage([authorization.canAccessFlags], async () => {
   const { t } = await serverTranslation("admin-flags");
 
+  // Get and sync user feature flags for all users
+  const usersWithFeatures = await getAllUsersWithFeatures();
+  await syncUserFeatureFlagsToRedis(usersWithFeatures);
```

Comment:
why sync to redis here?

Reply:
Redis doesn't need a complete list of user features, user permissions get synced to Redis when they login, as needed.

Reply:
Right but to flip the question, is there there any reason not to populate the Redis case with all user features?

I'm not tied to this though. I see it as more of a nice to have.

Reply:
Actually just to get this done, I'll remove that part of the code. Easy to add it back later if we decide to